### ISSUE TITLE
Issues with cite commands when using descending numbering

### ISSUE THREAD

**Issue by nplatis** [jspitz/biblatex-publist#54]
I discovered several issues with the cite commands when using (local or global) descending numbering.

1. Commands `\citesecitem*` and `\citesecitem` produce `[???]` with both `global-descending` and `local-descending` for `plnumbering`.
2. Command `\citeitem` produces wrong numbers only for the first section with `global-descending` numbering.

Here is a file to demonstrate the problems:

```
% !BIB program = biber

\documentclass{article}

\begin{filecontents}[nosearch,overwrite]{journals.bib}
@ARTICLE{j1,
  author = {Author, A.},
  title = {My 1st journal article},
  journal = {A journal},
  year = {2001},
}

@ARTICLE{j2,
  author = {Author, B.},
  title = {My 2nd journal article},
  journal = {Another journal},
  year = {2002},
}
\end{filecontents}

\begin{filecontents}[nosearch,overwrite]{conferences.bib}
@INPROCEEDINGS{c1,
  author = {Author, C.},
  title = {My 1st conference paper},
  booktitle = {A conference},
  year = {2011},
}

@INPROCEEDINGS{c2,
  author = {Author, D.},
  title = {My 2nd conference paper},
  booktitle = {Another conference},
  year = {2012},
}
\end{filecontents}

\begin{filecontents}[nosearch,overwrite]{books.bib}
@BOOK{b1,
  author = {Author, E.},
  title = {My 1st book},
  year = {2021},
}
  
@BOOK{b2,
  author = {Author, F.},
  title = {My 2nd book},
  year = {2022},
}
\end{filecontents}

\usepackage{csquotes}% not required, but recommended

\usepackage[
  style=publist,
  plnumbering=global,
]{biblatex}

\addbibresource{journals.bib}
\addbibresource{conferences.bib}
\addbibresource{journals.bib}


\begin{document}

\section{Main text}

Let's cite something:\\
\texttt{citesecitem* j1:} \citesecitem*{j1}\\
\texttt{citesecitem* c1:} \citesecitem*{c1}\\
\texttt{citesecitem j1:} \citesecitem{j1}\\
\texttt{citesecitem c1:} \citesecitem{c1}\\
\texttt{citeitem j1:} \citeitem{j1} (should not work)\\
\texttt{citeitem c1:} \citeitem{c1} (should not work)\\
\texttt{cite j1:} \cite{j1} (should not work)\\
\texttt{cite c1:} \cite{c1} (should not work)\\


\section{Publications}

\subsection{Journals}
\newrefsection[journals]
\texttt{citeitem j1:} \citeitem{j1}\\
\texttt{citeitem j2:} \citeitem{j2}\\
\nocite{*}
\printbibliography[heading=none]


\subsection{Conferecnes}
\newrefsection[conferences]
\texttt{citeitem c1:} \citeitem{c1}\\
\texttt{citeitem c2:} \citeitem{c2}\\
\nocite{*}
\printbibliography[heading=none]

\subsection{Books}
\newrefsection[books]
\texttt{citeitem b1:} \citeitem{b1}\\
\texttt{citeitem b2:} \citeitem{b2}\\
\nocite{*}
\printbibliography[heading=none]

\end{document}
```

**Comment by jspitz** [comment]
I cannot follow. I am getting:
![grafik](https://github.com/user-attachments/assets/44540579-d5fc-48a2-91b6-ee9cd482982c)

which looks correct. Did you run LaTeX often enough?

**Comment by jspitz** [comment]
Oh, I see, taking a look ...

**Comment by jspitz** [comment]
>     1. Commands `\citesecitem*` and `\citesecitem` produce `[???]` with both `global-descending` and `local-descending` for `plnumbering`.

This one should be fixed. This actually wasn't supported yet.

**Comment by jspitz** [comment]
The second issue should be fixed as well (at 215433a2d2b4).
